README.md

```diff
@@ -1,148 +1,211 @@
+It is to be expected that one digital service provider might own and provide services through multiple domains. In this case, you can maintain a single carbon.txt at one domain (e.g. _<https://my-org.com/.well-known/carbon.txt>_) and refer other domains to that single source of truth. To do this:
 
-_Anything that can follow links, and check a name against a register can confirm these claims are being made too_
+1. Complete the steps above to register as a provider, and create a carbon.txt file.
+2. **Generate a domain hash for your account** \
+   To do this, use the Observable Notebook linked below: \
+   [https://observablehq.com/d/17c922a33a5d6867](https://observablehq.com/d/17c922a33a5d6867) \
+   (Note: We understand this is solution is not ideal & are working on a nicer way to generate secrets)
+3. **Set DNS records for the domains you want to link back to your main carbon.txt** \
+   You can now create DNS TXT records for all the domains you want to link back to your main carbon.txt file. \
+   For example: _my-org.com_ also owns _me.my-org.com_. In order to link _me.my-org.com_ to the main carbon.txt file, they would create a TXT record that looks something like:
 
-## What if a company isn't running _entirely_ on green power?
+   ```DNS
+   TXT me.my-org.com carbon-txt=<https://my-org.com/.well-known/carbon.txt> generated_domain_hash
+   ```
 
-It's currently rare for companies _fully_ run all their infrastructure on green power. But maybe it's worth recognising when they do, to give credit when they do.
+#### For digital service providers who want to pass on their green claims to users of their service
 
-Let's take our Netwise example. They have a green data centre in London, but they _don't_ make claims about the rest of their datacentres, so we assume the default grey power there.
+Providers may want to broadcast that all the websites using their service are also running on green energy. To do this, they can use the [HTTP Via header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/via).
 
-It seems reasonable to expect Krystal to ask Netwise, and ask which datacentre they use for green power, and them to respond with name for it like 'london'. 
+1. Complete the steps above to register as a provider, and create a carbon.txt file.
+2. **Generate a domain hash for your account** \
+   To do this, use the Observable Notebook linked below: \
+   [https://observablehq.com/d/17c922a33a5d6867](https://observablehq.com/d/17c922a33a5d6867) \
+   (Note: We understand this is solution is not ideal & are working on a nicer way to generate secrets)
+3. **Set via headers for on requests generated/processed by your service** \
+   You can now set via headers for requests that go through your service. This allows you to broadcast that the request has come from your service, and point to your carbon.txt file. A valid via header would look like:
```

Comment:
`requests` should be `response`, I guess, since the service will respond with a via header with reference to the carbon.txt

Reply:
Thanks for picking that up @jonarnes